auth-web/src/components/auth/BaseAddress.vue

```diff
@@ -68,6 +68,12 @@
         </v-col>
       </v-row>
     </fieldset>
+    <div class="value value__title" aria-labelledby="mailingAddress" v-if="address" v-can:VIEW_ADDRESS.show>
```

Comment:
oh..i missed this from last PR as well.. the v-can doesnt have a SHOW I guess.. so have to revisit this usage...
we can discuss this and may be add show to the v-can

Reply:
Changed to hide